assets/scss/custom.scss

```diff
@@ -342,22 +344,19 @@ footer {
       padding: 1rem;
     }
 
-    th {
-      background-color: $white;
-    }
-
-    caption, tr {
-      background-color: $table-row-default;
-    }
+    &.object-table, &.response-table {
+        caption, tbody tr {
+            background-color: $table-row-default;
```

Comment:
table rows now blend in with the background:

![image](https://user-images.githubusercontent.com/1190097/182491164-559116e9-bf2d-4e72-b9c1-2ee69fbc9368.png)

Reply:
I'd suggest making `$table-row-alternate` be `white` instead, as it makes `#f4fafc` pop more. Alternatively, don't bother with striping and play around with table layout some more, though this is more involved and requires a bunch of awkward CSS to get right (we'd want to change various alignments and justifications, possibly copy changes as well).

Reply:
Yeah, I'd noticed that, but decided I would rather live with it than have the alternate table rows be *lighter* than the background, which looks really odd to me.

How about we give the table a light border? Done in b62be510 - I think it looks quite good, though ymmv

Reply:
![image](https://user-images.githubusercontent.com/1190097/182670435-e0ec2d0e-3762-4d78-8d27-001d22c62153.png)

it feels a bit like the table is now bleeding into the background instead of being the background, but also a border is a marked enough improvement for now. Typically we avoid table borders as it can feel overly restricting, though here it seems to fit well enough to break up the pages and pages of text.

Feels like a thing someone else can PR if they don't like it, now.